src/components/NavBar/store.js

```diff
@@ -1,20 +1,80 @@
-import { observable, action } from 'mobx';
+import { observable, action, computed } from 'mobx';
+import { OracleStatus, Token } from 'constants';
+import _ from 'lodash';
+import Topic from '../../stores/models/Topic';
+import Oracle from '../../stores/models/Oracle';
+
+import { queryAllTopics, queryAllOracles, queryAllVotes } from '../../network/graphQuery';
 
 const INIT_VALUES = {
   myActivitesCount: 0, // INIT_VALUESial loaded state
 };
 
 
-export default class NavBarStore {
-  @observable myActivitesCount = INIT_VALUES.myActivitesCount
+export default class {
+  @observable ResultSettingCount = INIT_VALUES.ResultSettingCount
+  @observable FinalizeCount = INIT_VALUES.FinalizeCount
+  @observable WithdrawCount = INIT_VALUES.WithdrawCount
+  @computed get totalCount() {
+    return this.ResultSettingCount + this.FinalizeCount + this.WithdrawCount;
+  }
 
-  @action.bound
-  async init() {
-    // this.myActivitesCount = await this.getMyActivitiesCount();
+  constructor(app) {
+    this.app = app;
   }
 
-  // @action.bound
-  // async getMyActivitiesCount() {
+  @action
+  init() {
+    this.getCount();
```

Comment:
you are missing the `reactions` for calling `getCount()` again on `syncBlockNum` and `walletAddresses` change. see `globalHub.js L83`.

Reply:
will updated